### ISSUE TITLE
Comma or semi-colon should generate the tag for searching in the browse view

### ISSUE THREAD

**Issue by jjchen20** [razzius/hms-weave#135]
In addition to the enter and tab buttons creating the tag for searching, typing a comma or semi-colon should also generate the tag in the search bar. This may help people realize that they can make lists of tags and the search will triangulate on people who fit all tag criteria.

**Comment by razzius** [comment]
Put an implementation of this on https://hms-weave-staging.herokuapp.com/login. As you'll see the menu stays open after a tag is created, which is a bit confusing. We can discuss relative importance of further improvements this evening.

**Comment by jjchen20** [comment]
Comma, semi-colon, enter, tab, and clicking away all create the tag now in staging, though the menu stays open only after comma and semi-colon creates the tag. It would be great if they were all the same and the menu closed after the tag is created.

**Comment by razzius** [comment]
Should be fixed on staging.

**Comment by JonKusner** [comment]
The tag searching works well in the main search bar, completing with both comma and semi colon. One cannot navigate with the arrow keys just like on the profile generation page. By contrast, in the degrees and institution sections, one may navigate with the arrow keys but comma and semi colon do not auto complete and one cannot delete tags by using the delete key an must instead manually exit out with the mouse

**Comment by jjchen20** [comment]
Likewise, same issue in my browser with the arrow keys not working for navigation.

**Comment by jjchen20** [comment]
A couple issues:
1) Navigation with arrow keys still does not work in the main search bar. 
2) Tag deletion for filtering by degrees and institutions in the "Browse all faculty profiles" view does not work when a) clicking the backspace button, or b) clicking on the "x" right by the tag. It only works when clicking the larger "X" to the right of the box.

**Comment by razzius** [comment]
Fixed the arrow keys. Opening a new issue for the tag deletion.
